src/www/ui/ui-clearing-view.php

```diff
@@ -217,7 +217,12 @@ function Output()
     $lastItem = GetParm("lastItem", PARM_INTEGER);
 
     if (!empty($lastItem)) {
-      $this->updateLastItem($userId, $groupId, $lastItem, $uploadTreeId);
+      if ($lastItem == $uploadTreeId) {
+        $currentUploadtreeId = $uploadTreeId;
+      } else {
+        $currentUploadtreeId = $lastItem;
+      }
+      $this->updateLastItem($userId, $groupId, $lastItem, $currentUploadtreeId);
```

Comment:
Can we simplify it saying `$currentUploadtreeId = $lastItem` and eliminate `$currentUploadtreeId` by passing `$lastItem` to `updateLastItem()`?